dashboard/app/models/pd/workshop.rb

```diff
@@ -683,6 +683,7 @@ def workshop_ending_date
   end
 
   def workshop_date_range_string
+    return I18n.t('not_applicable_abbreviation') if sessions.empty?
```

Comment:
Was this work from a different context?

Reply:
....weird. Yes, give me one second to fix